inference-engine/src/readers/ir_reader/ie_ir_parser.cpp

```diff
@@ -681,8 +681,14 @@ std::shared_ptr<ngraph::Node> V10Parser::XmlDeserializer::createNode(
     }
 
     // Try to create operation from loaded opsets
-    if (!ngraphNode && opsets.count(params.version)) {
-        auto opset = opsets.at(params.version);
+    auto version = params.version;
+    const std::set<std::string> experimental_detectrons = {"ExperimentalDetectronROIFeatureExtractor"};
```

Comment:
It can be static, and `unordered_set` is better for this situation

Reply:
Done.